google/cloud/security/scanner/samples/group_rules.yaml

```diff
@@ -1,50 +1,21 @@
-rules:
-  # Only allow Organization users to be in the Organization groups.
-  - name: sample whitelist for group users
-    mode: whitelist
-    resource:
-      - type: group
-        applies_to: self_and_children
-        resource_ids:
-          - my_customer
-    inherit_from_parents: true
-    members:
-      - role: MEMBER
-        type: USER
-        email: '*@YOUR_ORG_EMAIL_DOMAIN'
+- name: Allow my company users and gmail users to be in my company groups.
+  group_email: my_customer
+  mode: whitelist
+  conditions:
```

Comment:
Adding a TODO, as this kind of change will also required being baked ito the code.